src/nnet2/nnet-component.cc

```diff
@@ -4110,6 +4124,8 @@ void Convolutional1dComponent::Read(std::istream &is, bool binary) {
   ReadBasicType(is, binary, &patch_step_);
   ExpectOneOrTwoTokens(is, binary, ostr_beg.str(), "<PatchStride>");
   ReadBasicType(is, binary, &patch_stride_);
+  ExpectOneOrTwoTokens(is, binary, ostr_beg.str(), "<RearrangeInput>");
```

Comment:
I don't think we will have more than 2 types of vectorization, so the boolean would be sufficient. (Append descriptor will only for zyx or yzx vectorization)